docs/adrs/0275-custom-label-support.md

```diff
@@ -0,0 +1,43 @@
+# ADR 275: Custom Label Support
```

Comment:
Might want to change the title and file name to something like: `Support adding custom labels during runner config`
`Custom Label Support` is the overall name for custom runner labels feature we are doing which also include UI experience in github.com and documentation.

Reply:
Sure thing, i'll change it